routes/index.js

```diff
@@ -54,14 +54,15 @@ var routes = {
         amount: transaction.amount,
         stripeToken: transaction.stripeToken,
         email: transaction.email,
+        currency: transaction.currency,
         metadata: {
-          firstname: transaction.firstName,
-          lastname: transaction.lastName,
+          firstname: transaction.first,
```

Comment:
It's required, and I do create the customer inside the function I send this data to.

https://github.com/ScottDowne/donate.mozilla.org/blob/batmans-got-plans/routes/stripe.js#L22